README.md

```diff
@@ -40,6 +40,14 @@ yarn deploy
 
 ```
 
+> in a fourth terminal window:
+
+```bash
+
+yarn test
```

Comment:
Test doesn't do much here as part of the core setup, but it is definitely something cool to mention! Maybe this could go further down (as having it here implies that it is part of the basic lift-off)

Reply:
Well, for those of us who subscribe to the TDD philosophy, it *is* part of the basic lift-off ;-)  But I appreciate that there are others who don't.  It is already mentioned further down but it's buried in a long list under [the "Original Quickstart with TODO List" section](Original Quickstart with TODO List) which is doubly hidden because it's an `<h2>` nested inside `<h1>built with 🏗 scaffold-eth:</h1>`.  I assume this was unintentional and either this h2 should be an h1, or this h1 should be an h2, so that either way they are peer sections not parent/child.  But this points to a wider issue - IMHO the existing README is overwhelmingly long making it hard to follow the structure, and as such should be split into smaller pieces.  Please let me know how you want to proceed and I'll adjust this PR accordingly.

BTW I would have just dropped this contentious commit from the PR for the sake of getting the `--network hardhatevm` brokenness fixed ASAP, but then due to [bad design of GitHub's review system](https://github.com/isaacs/github/issues/998) this discussion thread would vanish.